doc/project/working_groups.rst

```diff
@@ -0,0 +1,117 @@
+TSC Working Groups
+******************
+
+Overview
+########
+
+The TSC, at its discretion, may establish working groups or subcommittees to
+serve as focused teams dedicated to specific technical areas, initiatives, or
+tasks.
+
+Membership
+##########
+
+Working Group Membership Eligibility
```

Comment:
I believe we must clarify the terms:

- Member / Membership
vs
- Core (team) member / Core (team) membership

Right now it is unclear to me whether those are two different categories or just a single one: is someone who participates in the meetings silently observing a member? Is membership to a WG just not meaningful, since they essentially amount to meetings with varied attendance? Should we rename "core (team) members" to something perhaps a bit clearer ("panel" or "council" or "board"?)

Reply:
you should be careful here about "membership". It feels like we are trying to define groups similar to IEEE where you end up defining how to be a member and what it means to retain membership and loose membership. If you start to go down this path then you also have to think through what it means if you give a WG certain responsibilities/powers if that group then becomes overly represented by a single company or small consortium of companies. 

IEEE WG usually has some sort of "membership" status occurring after <x> meetings of attendance out of <n> meetings (e.g. 3 meetings out of 5). You lose your membership after missing <y> meetings in a row (e.g. 3) and then have to go through the process of <x> out of <y> to regain your status. That doesn't mean you can not participate technically in a meeting it just means that when there are binding votes you status determines if you can vote. 

This also means there is no upper limit to how many can be part of the WG. The bias for who can be a member should be very lose (e.g. anyone). The exception would be in cases where there are ancillary requirements like the BLE work where you need to be a BT SIG member to be able to view advanced specs.

Reply:
@carlescufi  the "core" terminology was removed now.